README.md

```diff
@@ -281,9 +277,9 @@ That's it!
 
 ## More Detailed Documentation
 
-- **[All Rectors Overview](/docs/AllRectorsOverview.md)**
-- [How Rector Works?](/docs/HowItWorks.md)
-- [Nodes Overview](/docs/NodesOverview.md)
+- **[All Rectors: Overview](/docs/AllRectorsOverview.md)**
+- [How Does Rector Work?](/docs/HowItWorks.md)
+- [Nodes: Overview](/docs/NodesOverview.md)
 
```

Comment:
It's really "Nodes Overview" aka "Overview of Nodes", better keep it as there is just this one page about nodes.